retail/interactive-tutorial/RetailSearch.Samples/SearchSimpleQuerySample.cs

```diff
@@ -17,6 +17,8 @@
 using System.Collections.Generic;
 using System.Linq;
 
+// [START retail_search_simple_query]
```

Comment:
Please include the using statements within the region tags. (Same as I requested before)